### DISCUSSION TITLE
@antv/s2-react动态更改dataCfg里的column时,会报错_e/clearInitColumnLeafNodes is not a function

### DISCUSSION THREAD

**Discussion by dr0611** [antvis/S2#2724] [Q&A]
@antv/s2-react动态更改dataCfg里的column时,会报错_e/clearInitColumnLeafNodes is not a function

**Comment by lijinke666** [comment]
已解决: `@antv/s2-react` 和 `@antv/s2` 需要保证版本一致, 不能一个是 `1.x`, 一个是 `2.x`
